OrbitGl/TimeGraph.cpp

```diff
@@ -189,6 +189,24 @@ void TimeGraph::ZoomTime(float a_ZoomValue, double a_MouseRatio) {
   SetMinMax(minTimeUs, maxTimeUs);
 }
 
+void TimeGraph::VerticalZoom(float zoom_value, double mouse_ratio) {
+  static double increment_ratio = 0.1;
+
+  // In this way, VerticalZoom is reversible
```

Comment:
Can you add more of an explanation what "reversible" here means?

Reply:
The comment was only for reviewers, I'm going to erase it.
In the horizontal zoom, we use (1 + increment_ratio) and (1 - increment_ratio). These aren't reversibles (if you zoom-in and after zoom-out you aren't getting the initial ratio). So, after it, I want to change the other one also.

Reply:
Anyway, both changes (+ some refactoring in horizontal zoom) will be in another PR.